config/app.php

```diff
@@ -167,6 +167,7 @@
          * Package Service Providers...
          */
         Laravel\Tinker\TinkerServiceProvider::class,
+        Barryvdh\LaravelIdeHelper\IdeHelperServiceProvider::class,
```

Comment:
I prefer registering it in the AppServiceProvider after checking if it is a production environment.